lib/paperclip/processor.rb

```diff
@@ -36,7 +36,9 @@ def self.make file, options = {}, attachment = nil
     # The convert method runs the convert binary with the provided arguments.
     # See Paperclip.run for the available options.
     def convert(arguments = "", local_options = {})
-      Paperclip.run('convert', arguments, local_options)
+      page_number = local_options.delete(:page_number)
+      command = page_number ? "convert[#{page_number}]" : 'convert'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.